wcs-functions.php

```diff
@@ -49,20 +49,24 @@ function wcs_is_subscription( $subscription ) {
 }
 
 /**
- * A very simple check. Basically if we have ANY subscriptions in the database, then the user has probably set at
- * least one up, so we can give them the standard message. Otherwise
+ * Return true if there are any subscriptions in the database (active or inactive).
```

Comment:
```suggestion
 * Returns true if there are any subscriptions in the database (active or inactive).
```

To follow [WP documentation language standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#language), the comment should be in the third-person singular format. 

So this could be as simple as plural `Returns` (see example above). Some other alternatives might be:

```php
/**
 * Checks if there are any subscriptions in the database (active or inactive).
```

```php
/**
 * Determines if there are any subscriptions in the database (active or inactive).

Reply:
Thanks! TIL, we could link to this from our handbook or docs maybe.

Reply:
Fixed up (and the bool/return doc) in bb9458df495717d29cd97d2e781a8033b6e206e4